source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -100,6 +103,22 @@ public IEnumerable<PimsAcquisitionOwner> GetOwners(long id)
             return _acquisitionFilePropertyRepository.GetOwnersByAcquisitionFileId(id);
         }
 
+        public PimsProject GetProject(long id)
```

Comment:
same thing here, can likely remove this and use the project information from the file instead.

Reply:
removed